src/secret/ledger_nano.rs

```diff
@@ -95,160 +114,217 @@ impl SecretManage for LedgerSecretManager {
     }
 }
 
+/// needs_blind_signing
+/// the Ledger Nano S(+)/X app can present the user a detailed view of the transaction before it
+/// is signed but only with BasicOutputs, without extra-features and if the Essence is not too large.
+/// If criteria are not met, blind signing is needed.
+/// This method finds out if we have to switch to blind signing mode.
+pub fn needs_blind_signing(prepared_transaction: &PreparedTransactionData, buffer_size: usize) -> bool {
```

Comment:
Does this need to be public ?

Reply:
I think we want to use it in wallet.rs so Firefly can know what it needs to display (full tx or just essence hash) for verification